scripts/change-log-generator-utils.js

```diff
@@ -0,0 +1,274 @@
+const constants = require('./change-log-constants');
```

Comment:
dumb q: why do we write our script in straight JS instead of TS?

Reply:
To save an extra step to compile? I assume this was started when our release process was simple, but over time it grew and now I too felt having them in TS would help.

Reply:
cool.  I'll create a ticket.  We can use ts-node to run them as well so compile happens on the fly